CMakeLists.txt

```diff
@@ -628,6 +643,10 @@ if(BUILD_STUB)
 
 endif()
 
+if(BUILD_WITH_WREN)
```

Comment:
@nesbox take a look at here. the TIC_BUILD_WITH_WREN is not removed, but moved here, exactly following what you said in the other comment.. You also see mruby and janet flags defined in the same way below.